enterprise-ws/2.0/_index.md

```diff
@@ -1,21 +1,24 @@
 ---
-title: "Jakarta Enterprise Web Services 2.0 (milestone)"
-date: 2020-06-05
+title: "Jakarta Enterprise Web Services 2.0"
+date: 2020-09-05
 summary: "Release for Jakarta EE 9"
 ---
 
 This specification defines the Web Services for Jakarta EE architecture.
 
 * [Jakarta Enterprise Web Services 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jaxws/releases/2.0-enterprise-web-services)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Enterprise Web Services 2.0 Specification Document](./enterprise-ws-spec-2.0-RC1.pdf) (PDF)
-* [Jakarta Enterprise Web Services 2.0 Specification Document](./enterprise-ws-spec-2.0-RC1.html) (HTML)
+* [Jakarta Enterprise Web Services 2.0 Specification Document](./enterprise-ws-spec-2.0.pdf) (PDF)
+* [Jakarta Enterprise Web Services 2.0 Specification Document](./enterprise-ws-spec-2.0.html) (HTML)
+* Jakarta Enterprise Web Services 2.0 XML Schemas
+  * [Web Services deployment descriptor](https://jakarta.ee/xml/ns/jakartaee/jakartaee_web_services_2_0.xsd)
+  * [Web Services Client deployment descriptor](https://jakarta.ee/xml/ns/jakartaee/jakartaee_web_services_client_2_0.xsd)
 * [Jakarta Enterprise Web Services 2.0 TCK]()([sig](),[sha](),[pub]())
```

Comment:
- [ ] EFTL TCK link of the form http://download.eclipse.org/.../+.zip

In the PR description, the TCK it's indicated with `N/A` since it's covered by platform TCK. But the `/_index.md` still shows the bullet for TCK. Is this intended or the bullet should be removed?
![image](https://user-images.githubusercontent.com/1454523/88435021-9d9cca00-cdbe-11ea-91f6-39969d134cd7.png)